packages/schematics/angular/guard/index_spec.ts

```diff
@@ -105,4 +105,17 @@ describe('Guard Schematic', () => {
       expect(fileString).toContain(functionName);
     });
   });
+
+  it('should use CanActivate if no implements value', async () => {
+    const options = { ...defaultOptions, implements: []};
```

Comment:
If the user hits enter before selecting, is the value an empty array or undefined?

In that case, probable we'd need to handle empty array + undefined.

Reply:
The value is an empty array, and the undefined case was already handled by assigning an empty array by default 👍

Reply:
So the default empty array assignment is a problem. In general, it's preferred to use the schema defaults.

Reply:
The schema default gets applied prior of the schematic code execution. Meaning that if `option-x` has a default within the schematic it cannot be undefined.